docs/contributing/testing.md

```diff
@@ -21,11 +21,18 @@ Fix any CSS or JavaScript style errors that the linting tests report.
 
 ## 2. Fix any invalid HTML
 
-To check whether your HTML markup is valid, [check your page with W3C's markup validator](https://validator.w3.org/).
+To make sure your HTML markup is valid, [check your page with W3C's markup validator](https://validator.w3.org/). The validator checks your document's source code. To see your document's source code, select [View Source](https://developer.mozilla.org/en-US/docs/Tools/View_source) in your browser.
```

Comment:
(Q1) I think this doc is meant to be relatively high-level - this section is one of several places where we could nest further procedures, but the doc is more of a quick guide than an inventory of everything a user could do. Also, our feeling is that we don't want go into overmuch detail about things an audience of experienced developers is likely to know already. (Kind of like how our macros content stops short of explaining basic HTML stuff.) The link is intended to aid any less experienced users (presumably a small minority) who might not have the same level of knowledge.

(2) The user need came from a cross-gov Slack thread where an accessibility specialist mentioned the markup validator. Some developers asked to learn more, and one of them felt it would be a good thing to mention in this doc.

Reply:
Cheers Eoin - I'm still wondering if we need any of the detail after the link itself ... re Q2 - it's definitely a useful thing to include the markup validator but I still don't think we need to explain how it works when the user can click on the link to see how it works on W3C's site, if that makes sense?

Reply:
It does! How does this look?

```suggestion
To make sure your HTML markup is valid, [check your page with W3C's markup validator](https://validator.w3.org/).
```

Reply:
```suggestion
To make sure your HTML markup is valid, [check your page with W3C's markup validator](https://validator.w3.org/).
```